debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MysqlDefaultValueIT.java

```diff
@@ -534,6 +536,8 @@ public void dateAndTimeTest() throws InterruptedException {
         assertThat(schemaH.defaultValue()).isEqualTo(82800700000L);
         assertThat(schemaI.defaultValue()).isEqualTo(82800123456L);
 
+        assertThat(schemaL.defaultValue()).isEqualTo(-82800700000L);
```

Comment:
I'll change it to give the expected values as `Duration`, making this more readable.